src/components/Games/QuestionFrame/Choices/Choices.css

```diff
@@ -18,11 +18,18 @@
 }
 
 .choice-button-small {
```

Comment:
can you add prettier and other code formatters to VSCode? this will make sure that your code is formatted in the same way as mine